String pad (#7340)
fish-shell/fish-shell: doc_src/cmds/string-pad.rst

string-pad - pad characters before and after string
===================================================

Synopsis
--------

.. BEGIN SYNOPSIS

::

    string pad [(-r | --right)] [(-c | --char) CHAR] [(-w | --width) INTEGER] [STRING...]

.. END SYNOPSIS

Description
-----------

.. BEGIN DESCRIPTION

> ``string pad`` pads specified character before and after the string for each STRING.

Reviewer: ```suggestion
``string pad`` pads each STRING with CHAR to the given width.
```

> The default behaviour is left padding with spaces and default width is the length of string (hence, no padding).

Reviewer: I guess we mostly use American English `behavior`

Author: My bad, fixed. 👍

If ``-r`` or ``--right`` is given, only pad after string.

> The ``-c`` or ``--char`` switch causes the characters in CHAR to be padded instead of default whitespace character.

Reviewer: `characters` is slightly confusing, it's only one character

```suggestion
The ``-c`` or ``--char`` switch causes padding with the character CHAR instead of default whitespace character.
```

> If ``-w`` or ``--width`` is given, pad the string to given length. Width less than the string length will result in an unchanged string.

Reviewer: ```suggestion
If ``-w`` or ``--width`` is given, pad the string to given width. Width less than the string width will result in an unchanged string.
```

> Exit status: 0 if string was padded or haven't changed, or 1 otherwise.

Reviewer: I think this line can be removed since that's no longer the case.
